### ISSUE TITLE
Wrong indentation when using conditional comments

### ISSUE THREAD

**Issue by fcalderan** [jonschlinkert/grunt-prettify#2]
Since I'm using HTML5 boilerplate, my html file starts with

```
<!DOCTYPE html>
<!--[if IEMobile 7 ]><html class="no-js iem7" lang="it"><![endif]-->
<!--[if lt IE 9]><html class="no-js lte-ie8" lang="it"><![endif]-->
<!--[if (gt IE 8)|(gt IEMobile 7)|!(IEMobile)|!(IE)]><!--><html class="no-js" lang="it"><!--<![endif]-->
```

but after I run prettify task (with `indent_space : 2`)  my markup starts with

```
<!DOCTYPE html>
<!--[if IEMobile 7 ]>
  <html class="no-js iem7" lang="it">
  <![endif]-->
  <!--[if lt IE 9]>
    <html class="no-js lte-ie8" lang="it">
    <![endif]-->
    <!--[if (gt IE 8)|(gt IEMobile 7)|!(IEMobile)|!(IE)]>
      <!--><html class="no-js" lang="it"><!--<![endif]-->

```

How can I leave the conditional comments part untouched, so my `html` tag has no indent spaces at the beginning? 

Thank you.

**Comment by jonschlinkert** [comment]
I noticed the same on my projects, agreed that it's not very... ahem...pretty. 

I found a related issue on the [js-beautify](https://github.com/einars/js-beautify/issues) repo, which is the engine used by this plugin: https://github.com/einars/js-beautify/issues/91. 

Looks like it's been sitting for a while, I'll go add my +1, since I think this really needs to be resolved there. But I'll leave this open since it effects users of this plugin as well.
